_docs/adding-datasets.md

```diff
@@ -62,6 +62,9 @@ $ cat 01-head.xml dataset1.xml dataset2.xml tail.xml > datasets.xml
 $ cp datasets.xml erddap/content/datasets.xml
 ```
 ### Use Python's [`xml.etree.ElementTree`](https://docs.python.org/3/library/xml.etree.elementtree.html) package to insert elements into a template:
+
+**Do note that some sections of the output of `GenerateDatasetsXml.sh` are commented with the `<!--` and `-->` characters. This approach will remove such sections.**
```

Comment:
@srstsavage please check if this is better. From my understanding, all of the `<sourceAttributes>` tags are commented in the output of `GenerateDatasetsXml.sh`? IIRC at one point I had them on `datasets.xml` and ERDDAP complained. It will only accept the `<addAttributes>` tag. Please correct me if I'm wrong.

Thanks for the review :)

Reply:
Yes, very clear now. Thank you!